verification/src/http_server/handlers/verification/solidity/types.rs

```diff
@@ -17,20 +22,7 @@ pub struct VerificationResponse {
     pub verified: bool,
 }
 
-#[derive(Debug, Deserialize, PartialEq)]
-#[serde(rename_all = "camelCase")]
-enum EvmVersion {
-    Homestead,
-    TangerineWhistle,
-    SpuriousDragon,
-    Byzantium,
-    Constantinople,
-    Petersburg,
-    Istanbul,
-    Berlin,
-    London,
-    Default,
-}
+type EvmVersion = ethers_solc::EvmVersion;
```

Comment:
`ethers_solc::EvmVersion` contains items starting with capital letters. we have to rename all of them to camel case. before it was done by using `#[serde(rename_all = "camelCase")]` but I don't know how to do it without access to struct

Reply:
yes, I know, I'll fix it